src/netspresso_trainer/models/op/custom.py

```diff
@@ -613,19 +613,114 @@ def __init__(
         out_channels,
         shortcut=True,
         expansion=0.5,
-        #depthwise=False,
+        depthwise=False,
+        norm_type: Optional[str] = "batch_norm",
         act_type="silu",
+        no_out_act=False,
+        is_stem_stage=False,
     ):
+
         super().__init__()
-        hidden_channels = int(out_channels * expansion)
-        self.conv1 = ConvLayer(in_channels=in_channels, out_channels=hidden_channels,
-                                kernel_size=1, stride=1, act_type=act_type)
-        self.conv2 = ConvLayer(in_channels=hidden_channels, out_channels=out_channels,
-                                kernel_size=3, stride=1, act_type=act_type)
+
+        if is_stem_stage:
+            hidden_channels = round(in_channels * expansion)
+        else:
+            hidden_channels = round(out_channels * expansion)
```

Comment:
Why is_stem_stage is added here?
Can you explain the purpose of these lines?

Reply:
I switch back the round function with int() updated the model config file to make it correctly compute the number of hidden channels.